### ISSUE TITLE
Implement user interface

### ISSUE THREAD

**Issue by SeanArchibald** [xenharmonic-devs/scale-workshop#18]
This is the parent issue for all UI tasks.

- [x] Meta tags in index.html head
- [x] #22
- [x] Status bar (e.g. for 'keyboard enabled' indicator)
- [x] Logo
- [x] Favicon setup in index.html head

### Common UI classes
- [x] Button groups
- [x] Control groups
- [x] Drop-down menus
- [x] Modals

### Scale Workshop tab

- [x] About

### Scale Edit tab

- [x] Scale data
- [x] Frequency/mapping
- [x] Export tuning
- [x] Tuning table

### Analysis tab

- [x] Analysis tab

### Virtual keyboard tab

- [x] Virtual keyboard

### Synth tab

- [x] Synth parameters
- [x] Quiet/panic

### MIDI I/O tab

- [x] MIDI I/O parameters

### Preferences tab

- [x] Preferences

**Comment by SeanArchibald** [comment]
Ready to close this once my new branch status-tray is merged into main
